compiler/dyno/lib/uast/BuilderResult.cpp

```diff
@@ -115,13 +117,28 @@ void BuilderResult::mark(Context* context) const {
 
   // mark UniqueStrings in the Locations
```

Comment:
Might be nice to add a comment that much of this is probably redundant (the set of IDs will be covered by only a few of these calls).